pkg/minikube/driver/install.go

```diff
@@ -133,6 +132,7 @@ func validateDriver(executable string, v semver.Version) (string, error) {
 	if err != nil {
 		return path, errors.Wrap(err, "can't parse driver version")
 	}
+
```

Comment:
Now that we allow version mismatches, could you please add a log sttatement declaring what driver version we are using? This will help for future debugging;

`klog.Infof("%s version is %s", path, driverVersion)`

Thanks!

Reply:
Sure. Done.